src/traces/pie/defaults.js

```diff
@@ -65,19 +65,17 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
         }
     }
 
+    handleDomainDefaults(traceOut, layout, coerce);
+
+    var hole = coerce('hole');
     var title = coerce('title');
     if(title) {
         var titlePosition = coerce('titleposition');
```

Comment:
I still think we should make the default be `'middle center'` when there's a `hole` (ie `var titlePosition = coerce('titleposition', hole ? 'middle center' : 'top center');` and remove `attributes.titleposition.dflt`) - or maybe when `hole > 0.5` or something... would be strange to put the title in the center when the hole is very small!

But I'm not wedded to that idea; if you've considered it and still think `'top center'` should be the default in all cases we can keep it that way.